test/spec/modules/33acrossIdSystem_spec.js

```diff
@@ -192,6 +192,185 @@ describe('33acrossIdSystem', () => {
       });
     });
 
+    context('when screen and viewport height and width are defined', () => {
+      it('should call endpoint with sch, scw, vpw, vph', () => {
+        const completeCallback = () => {};
+        const { callback } = thirthyThreeAcrossIdSubmodule.getId({
+          params: {
+            pid: '12345'
+          }
+        });
+
+        const win = {
+          parent: null,
+          devicePixelRatio: 2,
+          screen: {
+            width: 1024,
+            height: 728,
+            availHeight: 500
+          },
+          navigator: {
+            maxTouchPoints: 0
+          },
+          document: {
+            visibilityState: 'visible',
+            documentElement: {
+              clientWidth: 800,
```

Comment:
See my comment on the bid adapter recommending to use different test fixture values for clientWidth & clientHeight.